api/routes/slideshow.js

```diff
@@ -26,14 +27,47 @@ router
   })
   .delete('/:id', (req, res, next) => {
     const { id } = req.params
-    return Slideshow.findById(id)
+    return Slideshow.findByIdAndDelete(id)
       .then(slideshow => {
         if (!slideshow) return next('Slideshow not found')
-        return slideshow.remove().then(() => {
-          return res.send('success')
-        })
+        /**
+         * return slideshow.remove().then(() => {
```

Comment:
Don't leave code uncommented, just remove it. If there's a problem we'll go back to the git history.